playbooks/roles/nginx/files/sites/media/media.plebia.org.conf

```diff
@@ -0,0 +1,42 @@
+server {
```

Comment:
@Agrendalath, I think we should keep all the non-OpenCraft stuff out of this repository.